src/basic/log.c

```diff
@@ -552,7 +552,7 @@ static int write_to_kmsg(
         if (kmsg_fd < 0)
                 return 0;
 
-        if (!ratelimit_below(&ratelimit))
+        if (log_get_max_level() != LOG_DEBUG && !ratelimit_below(&ratelimit))
```

Comment:
IIRC, this had caused a serious problem: https://bugs.freedesktop.org/show_bug.cgi?id=76935.

(Or the kernel can actually handle more logs now?)

Reply:
I think it's better to relax the ratelimit a bit, rather than disabling it completely 🤔

Reply:
As mentioned in the docs for this function, this is only intended to catch where we start writing logs in a tight loop, not for excessive logging, so it should be fine to disable in debug mode.